### PR TITLE
fix crashes

### PR DESCRIPTION
**Pull request by rettinghaus**
We found some minor issues that should be resolved with this PR.

### CODE REVIEW

File: src/view_beam.cpp [hunk 1]
```diff
@@ -116,6 +116,11 @@ void View::DrawFTrem(DeviceContext *dc, LayerElement *element, Layer *layer, Sta
     /******************************************************************/
     // Calculate the beam slope and position
 
+    // We reached here (so fTrem children are not empty and there are at least 2 beam coodrs)
+    // If for some reason coords refs are not initialized - do that here
```
Comment (lpugin) [inline src/view_beam.cpp:120]: Do you have any idea how we can end up in this situation?

Comment (rettinghaus) [reply src/view_beam.cpp:120]: It seems to be connected with fingered tuplet tremolos coming from MusicXML. Although I'm not totally sure what causes this problem.

Comment (lpugin) [reply src/view_beam.cpp:120]: Could you post a minimal MEI example?

Comment (rettinghaus) [reply src/view_beam.cpp:120]: @lpugin I brought it down to this example with a wrong `fTrem`: 
each measure for itself works, but in combination it crashes.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title></title>
         </titleStmt>
         <pubStmt><date isodate="2021-03-02">2021-03-02</date>
         </pubStmt>
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv xml:id="mdiv-0000000858131134">
            <score xml:id="score-0000000109795886">
               <scoreDef xml:id="scoredef-0000000530523754">
                  <staffGrp xml:id="staffgrp-0000000884127240">
                     <staffGrp xml:id="staffgrp-0000001087169087">
                        <grpSym xml:id="grpsym-0000001259976533" symbol="brace" />
                        <staffGrp xml:id="P1" bar.thru="true">
                           <staffDef xml:id="staffdef-0000001040942941" n="1" lines="5" ppq="480">
                              <clef xml:id="clef-0000002105110912" shape="G" line="2" />
                              <keySig xml:id="keysig-0000000806013659" sig="1f" />
                              <meterSig xml:id="msig-0000000344721537" count="4" sym="common" unit="4" />
                           </staffDef>
                        </staffGrp>
                     </staffGrp>
                  </staffGrp>
               </scoreDef>
               <section xml:id="section-0000000651003229">
                  <measure xml:id="measure-0000000769347005">
                     <staff xml:id="staff-0000001563585622" n="1">
                        <layer xml:id="layer-0000000795157582" n="9">
                           <tuplet xml:id="tuplet-0000002041368145" num="2" numbase="1" bracket.visible="true">
                              <fTrem xml:id="ftrem-0000001075668543" beams="2" beams.float="2">
                                 <chord xml:id="chord-0000001978696387" dots="1" dur="2" stem.dir="down">
                                    <note xml:id="note-0000001243861755" oct="3" pname="f" />
                                    <note xml:id="note-0000000018418867" oct="3" pname="a" accid.ges="f" />
                                 </chord>
                              </fTrem>
                           </tuplet>
                        </layer>
                     </staff>
                  </measure>
                  <measure xml:id="measure-0000001863939266">
                     <staff xml:id="staff-0000001773657780" n="1">
                        <layer xml:id="layer-0000001709782098" n="9">
                           <tuplet xml:id="tuplet-0000000829040579" num="2" numbase="1" bracket.visible="true">
                              <fTrem xml:id="ftrem-0000000811109517" beams="2" beams.float="2">
                                 <chord xml:id="chord-0000001735278236" dots="1" dur.ppq="480" dur="2" stem.dir="down">
                                    <note xml:id="note-0000000091461063" oct="3" pname="f" />
                                    <note xml:id="note-0000001993386192" oct="3" pname="a">
                                       <accid xml:id="accid-0000002096835744" accid="f" accid.ges="f" />
                                    </note>
                                 </chord>
                                 <chord xml:id="chord-0000000794345512" dots="1" dur.ppq="480" dur="2" stem.dir="down">
                                    <note xml:id="note-0000001370031178" oct="3" pname="f" />
                                    <note xml:id="note-0000001806670432" oct="3" pname="a" accid.ges="f" />
                                 </chord>
                              </fTrem>
                           </tuplet>
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>

```

Comment (lpugin) [reply src/view_beam.cpp:120]: I will look at it. As far as I can see this MEI is not valid.

Comment (lpugin) [reply src/view_beam.cpp:120]: I pushed a fix here https://github.com/rism-digital/verovio/pull/2085

You probably still want to fix the MusicXML import
image

Comment (rettinghaus) [reply src/view_beam.cpp:120]: Thanks, what's with the other two? Should I just drop e3cfeb33fc64f5ad0ad6915528653e971d2b58cb?

Comment (lpugin) [reply src/view_beam.cpp:120]: Yes, since now there are not fixing anything.

**Pull request merged**